src/oph/ehoks/palaute/tyoelama.clj

```diff
@@ -34,6 +54,12 @@
         (= 12 month) (LocalDate/of (inc year) 1 1)
         :else (LocalDate/of year (inc month) 1)))))
 
+(defn voimassa-loppupvm
+  "Given `voimassa-alkupvm`, calculates `voimassa-loppupvm`, which is currently
+  60 days after `voimassa-alkupvm`."
+  [^LocalDate voimassa-alkupvm]
+  (.plusDays voimassa-alkupvm 60))
```

Comment:
tän varmaan pitäis myös olla rajattu siihen enintään 60pv (vai oliko 90pv?) herätepvm:stä·